### PR TITLE
init-pants: Only use CI config file if it exists

### LINKED ISSUE #11: init-pants should not require a pants.ci.toml
Currently that is its default value for the `pants-ci-config` key, and it fails if that file doesn't exist.

We should probably allow that file to not to exist? At least if it wasn't explicitly set (do actions allow us to detect whether it was set explicitly?)

### PR DESCRIPTION
**Pull request by cognifloyd**
Fixes #11

### CODE REVIEW

File: init-pants/action.yaml [hunk 1]
```diff
@@ -105,8 +105,16 @@ runs:
     - name: Tell pants to use CI config
       if: ${{ inputs.pants-ci-config != '' }}
       shell: bash
+      env:
+        PANTS_CONFIG_FILES: ${{ inputs.pants-ci-config }}
       run: |
-        echo "PANTS_CONFIG_FILES=${{ inputs.pants-ci-config }}" >> ${GITHUB_ENV}
+        if [ -f ${PANTS_CONFIG_FILES} ] || [ "${PANTS_CONFIG_FILES}" != "pants.ci.toml" ]; then
```
Comment (benjyw) [inline init-pants/action.yaml:111]: Is there any way of telling if the input was set explicitly? Ideally we would distinguish between "user didn't specify any file" and "user specified pants.ci.toml" explicitly?

Comment (cognifloyd) [reply init-pants/action.yaml:111]: I suppose we could change the default to something like `DEFAULT` and then if `== "DEFAULT"` and `pants.ci.toml` exists, use that. Would that be better?

Comment (benjyw) [reply init-pants/action.yaml:111]: Oh yeah, that would work!

Comment (cognifloyd) [reply init-pants/action.yaml]: Changed. How does it look now?

**Review by benjyw — APPROVED** [review]
LGTM!

**Pull request merged**